src/main/java/edu/kit/ipd/crowdcontrol/objectservice/api/Router.java

```diff
@@ -0,0 +1,127 @@
+package edu.kit.ipd.crowdcontrol.objectservice.api;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.protobuf.Message;
+import edu.kit.ipd.crowdcontrol.objectservice.proto.Template;
+import spark.Route;
+import spark.Spark;
+import spark.servlet.SparkApplication;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static spark.Spark.before;
+import static spark.Spark.exception;
+
+/**
+ * Application entry point. Defines the REST API.
+ *
+ * @author Niklas Keller
+ */
+public class Router implements SparkApplication {
+    private Gson gson;
+    private TemplateResource templateResource;
+
+    /**
+     * Creates a new instance. Call {@link #init()} afterwards to initialize the routes.
+     */
+    public Router() {
+        this.gson = new GsonBuilder().setPrettyPrinting().create();
+        this.templateResource = new TemplateResource();
+    }
+
+    @Override
+    public void init() {
+        exception(BadRequestException.class, (exception, request, response) -> {
+            response.status(400);
+            response.type("application/json");
+            response.body(gson.toJson(new ErrorResponse("badRequest", exception.getMessage())));
+        });
+
+        exception(NotAcceptableException.class, (exception, request, response) -> {
+            response.status(406);
+            response.type("application/json");
+            response.body(gson.toJson(new ErrorResponse("notAcceptable", exception.getMessage())));
```

Comment:
There's support for both, except for error responses currently, right. Will write a special error responder.